arangodb-net-standard/AnalyzerApi/Models/AnalyzerProperties.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace ArangoDBNetStandard.AnalyzerApi.Models
+{
+    /// <summary>
+    /// Properties of an Analyzer
+    /// </summary>
+    public class AnalyzerProperties
+    {
+        public string Locale { get; set; }
```

Comment:
It would be nice to provide XML doc comments to clarify the properties listed here, if you have them. I couldn't find them on the online doc. Not a blocker.

Reply:
Additional analyzer documentation is available here: https://www.arangodb.com/docs/stable/analyzers.html
I've added some XML comments and also included links for some detailed descriptions.